traffic_ops/testing/api/v14/profiles_test.go

```diff
@@ -69,33 +69,33 @@ func CreateTestProfiles(t *testing.T) {
 			t.Errorf("could not GET profile with name: %s %v\n", pr.Name, err)
 		}
 		if len(profiles) == 0 {
-			t.Errorf("could not GET profile %++v: not found\n", pr)
+			t.Errorf("could not GET profile %+v: not found\n", pr)
```

Comment:
does ErrorF need `\n` I dont think it does

Reply:
Nope, not sure how I missed this one. Thanks, I will update it.

Reply:
Oh, I see. My regex was only looking for things ending in `\n")` and that closing paren blocked finding this form.